src/redux/redux-config.ts

```diff
@@ -1,25 +0,0 @@
-import * as actionTemplates from './actions';
-import chromeReducer, { chromeInitialState } from '.';
```

Comment:
There is the `on` event which leverages the middlewards. It can be and should done via jotai. https://github.com/Hyperkid123/insights-chrome/blob/master/src/chrome/create-chrome.ts#L100.

It was already done for the `APP_NAVIGATION` public event. You can follow similar approach.